src/utils/components/PolicyForm/DeleteInterfaceModal.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useNMStateTranslation } from 'src/utils/hooks/useNMStateTranslation';
+
+import {
+  ActionList,
+  ActionListItem,
+  Button,
+  ButtonVariant,
+  Modal,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { NodeNetworkConfigurationInterface } from '@types';
+
+import { capitalizeFirstLetter } from './utils';
+
+type DeleteInterfaceModalProps = {
+  closeModal: () => void;
+  isOpen?: boolean;
+  policyInterface: NodeNetworkConfigurationInterface;
+  onSubmit: (policyInterface: NodeNetworkConfigurationInterface) => void;
+};
+
+const DeleteInterfaceModal: React.FC<DeleteInterfaceModalProps> = ({
+  closeModal,
+  isOpen,
+  policyInterface,
+  onSubmit,
+}) => {
+  const { t } = useNMStateTranslation();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    onSubmit(policyInterface);
+    closeModal();
+  };
+
+  return (
+    <Modal
+      className="ocs-modal"
+      onClose={closeModal}
+      variant="medium"
+      position="top"
+      title={t('Delete Node network configuration policy')}
```

Comment:
This should be the resource name: `NodeNetworkConfigurationPolicy`

Reply:
Okay great. I'll create a different pr to fix this in other locations

Reply:
Actually, we must also add an `Interface` in this word. This modal is for the interface, not the policy itself. Soooo we've caught it